### PR TITLE
feat(WebexMember): indicate current user if in meeting

### PR DESCRIPTION
**Pull request by maloun96**
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-220758

https://user-images.githubusercontent.com/8277264/112468341-ccfe3f00-8d70-11eb-970d-c09da8e3d3ff.mov

### CODE REVIEW

File: src/components/WebexMember/WebexMember.jsx [hunk 1]
```diff
@@ -31,7 +32,10 @@ export default function WebexMember({
   return (
     <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-member`}>
       <WebexAvatar personID={personID} displayStatus={displayStatus} />
-      <div className="member-name">{`${firstName} ${lastName}`}</div>
+      <div className="member-name">
```
Comment (cipak) [inline src/components/WebexMember/WebexMember.jsx:35]: let's take out the subtitle from the `member-name` class, something like:
```html
<div.member>
    <avatar/>
    <div.description>
        <div.name/>
        <div.subtitle/>
    </div>
</div>
```
Feel free to use a better name than `description` if you find one.

**Comment by cipak** [comment]
@maloun96 please use `WebexMember` instead of `WebexMemberRoster` in the commit message

File: src/components/WebexMember/WebexMember.jsx [hunk 2]
```diff
@@ -22,6 +22,7 @@ export default function WebexMember({
   displayStatus,
 }) {
   const {firstName, lastName} = usePerson(personID);
+  const {ID} = useMe();
```
Comment (cipak) [inline src/components/WebexMember/WebexMember.jsx:25]: to align with other PRs on this file and minimize the merge conflicts, please use a `me` variable:
```js
const me = useMe();
```

File: src/components/WebexMember/WebexMember.scss [hunk 3]
```diff
@@ -12,6 +12,12 @@
     padding-left: 0.25rem;
   }
 
+  .member-subtitle {
```
Comment (cipak) [inline src/components/WebexMember/WebexMember.scss:15]: let's use `subtext`, like in the story, and no `member-` prefix, we are in the member component context anyway

File: src/components/WebexMember/WebexMember.jsx [hunk 4]
```diff
@@ -22,16 +22,21 @@ export default function WebexMember({
...
     .find((itemMember) => itemMember.personID === personID);
 
   const isMuted = member && member.muted;
+  const showMe = me.ID === personID && destinationType === DestinationType.MEETING;
 
   return (
     <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-member`}>
       <WebexAvatar personID={personID} displayStatus={displayStatus} />
-      <div className="member-name">{`${firstName} ${lastName}`}</div>
+      <div className="description">
```
Comment (lalli-flores) [inline src/components/WebexMember/WebexMember.jsx:36]: Is this `description` div necessary?
Can the subtext just be part of the `member-name` div?

**Review by lalli-flores — COMMENTED** [review]
Minor comment but otherwise almost there 👍🏼

File: src/components/WebexMember/WebexMember.jsx (hunk 4 above)
Comment (maloun96) [reply src/components/WebexMember/WebexMember.jsx:36]: We agree with the team to use this structure, we consider that member-name should contain the only the name, after that will come 'representor', 'you', 'organization name'

**Review by lalli-flores — APPROVED** [review]

File: src/components/WebexMember/WebexMember.jsx (hunk 4 above)
Comment (lalli-flores) [reply src/components/WebexMember/WebexMember.jsx:36]: Ok, we can keep the structure if the team thinks that's best. I'm not a fan of the class name, not sure it makes sense to me why it's called `description`? Maybe a better name would be `details`

**Comment by lalli-flores** [comment]
@maloun96, if you can fix the conflicts, we should be good to merge

**Comment by maloun96** [comment]
@lalli-flores Done!

**Comment by lalli-flores** [comment]
I'm sorry seems like conflicts are back 😭

**Pull request merged**